ignore_words.txt

```diff
@@ -2,3 +2,4 @@ lod
 byteorder
 flem
 parm
+tht
```

Comment:
I can't `git grep tht` anywhere. So why the ignore?

Reply:
That's my fault; I added this as my rst checker compiles on the fly and there are some temporary files that are being flagged by the spell check.  We can remove this.

Reply:
```suggestion
```